relay-server/src/actors/project.rs

```diff
@@ -153,9 +153,10 @@ impl ProjectState {
         self.disabled
     }
 
-    /// Returns `true` if the project state obtained from the upstream could not be parsed. This
-    /// results in events being dropped similar to disabled states, but can provide separate
-    /// metrics.
+    /// Returns whether the project state obtained from upstream could not be parsed.
```

Comment:
I find the old sentence much easier to understand.  And describes exactly what to expect. 

```suggestion
    /// Returns `true` if the project state obtained from upstream could not be parsed.
```